docs/contract-openapi-draft.yml

```
@@ -0,0 +1,398 @@
+openapi: 3.0.1
+info:
+  title: User Service
+  description: 'Example of a user management with its basic operations: create user, delete user, edit user by id, list users and get user by id.'
+  version: 1.0.0
+servers:
+  - url: http://{domain}:{server}/{basePath}
+    variables:
+      domain:
+        default: 'localhost'
+      server:
+        default: '8080'
+      basePath:
+        default: 'api/v1'
+tags:
+  - name: user
+    description: Operations to manage users
+paths:
+  '/users':
+    post:
+      tags:
+        - user
```

mismo comentario de arriba

```suggestion
        - User
```

---

La idea de implementar un spec para establecer el contrato de la api no es que se avance conforme a la implementacion de los endpoints?. Yo dejaría esta spec solo con el endpoint de creación